package.json

```diff
@@ -40,6 +40,9 @@
     "lint:styles:fix": "yarn run lint:styles --fix",
     "lint:scripts": "eslint --ignore-path .gitignore --ext=js,jsx,ts,tsx .",
     "lint:scripts:fix": "yarn run lint:scripts --fix",
+    "postinstall": "husky install",
+    "postpack": "pinst --enable",
+    "prepack": "pinst --disable",
```

Comment:
This pinst stuff is unfortunate, but Husky needs it now.

Apparently postinstall runs in consuming apps, as well. We don't want husky to install in other apps, so this disables that behavior or something.